src/WSRequestHandler_Sources.cpp

```diff
@@ -923,7 +923,7 @@ HandlerResponse WSRequestHandler::HandleGetBrowserSourceProperties(WSRequestHand
 	}
 
 	QString sourceId = obs_source_get_id(source);
-	if (sourceId != "browser_source") {
+	if (sourceId != "browser_source" && sourceId != "linuxbrowser_source") {
```

Comment:
It's `linuxbrowser-source`, not `linuxbrowser_source`: https://github.com/bazukas/obs-linuxbrowser/blob/master/src/plugin/main.c#L630